shopping_mall/backend/ai/agent/clients/claude.py

```
@@ -0,0 +1,100 @@
+                    )
+                elif block.type == "text" and block.text:
+                    text = block.text
+
+            return AgentResponse(
+                text=text if not tool_calls else None,
+                tool_calls=tool_calls,
+                raw=response,
+            )
+
+        except anthropic.AuthenticationError as e:
+            logger.error(f"Claude API 인증 실패: {e}")
```

Ruff (0.15.9)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)